crypter.go

```diff
@@ -403,9 +423,20 @@ func (ctx *genericEncrypter) Options() EncrypterOptions {
 	}
 }
 
-// Decrypt and validate the object and return the plaintext. Note that this
-// function does not support multi-recipient, if you desire multi-recipient
+// Decrypt and validate the object and return the plaintext. This
+// function does not support multi-recipient. If you desire multi-recipient
 // decryption use DecryptMulti instead.
+//
+// The decryptionKey argument most have one of these types:
+//   - *rsa.PrivateKey
+//   - *ecdsa.PrivateKey
+//   - []byte (will be interpreted as the bytes of a symmetric key)
+//   - string (will be interpreted as the bytes of a symmetric key)
```

Comment:
Here you list []byte and string in the middle of the set of acceptable types; on line 132 you listed them last.